.circleci/config.yml

```diff
@@ -50,16 +50,18 @@ jobs:
           name: Pull CARMAMsgs
           command: |
             source ${INIT_ENV}
-            git clone -b CARMAMsgs_1.1.0 --depth 1 git@github.com:usdot-fhwa-stol/CARMAMsgs.git src/CARMAMsgs
+            git clone -b develop --depth 1 git@github.com:usdot-fhwa-stol/carma-msgs.git src/CARMAMsgs
```

Comment:
Shouldn't this clone from master?

Reply:
Good point, should be updated with base image.